pkg/scheduler/core/pod_scheduler/pod_scheduler_test.go

```diff
@@ -61,6 +64,7 @@ func TestSelectHost(t *testing.T) {
 			},
 			possibleHosts: sets.NewString("machine2.1"),
 			expectsErr:    false,
+			podOwner:      "foo",
```

Comment:
Why use such meaningless variable names?

Reply:
I have changed them to some more meaningful names.

Reply:
pod owns represents a resource that owns it. Ex: deployment rs is not a specific person's name, which is meaningless.

Reply:
https://github.com/kubewharf/godel-scheduler/blob/main/pkg/util/pod/util.go#L696

Reply:
you mean something like "default/rs"?

Reply:
You can take a look at the source code. The source code uses "/" to distinguish resources and namesapce to represent the podOwn field. Maybe you can also use " " directly, because the original code uses " ". I'm not sure why we need to modify this variable. There are Any special purpose?

Reply:
It is because an empty podOwner will not trigger cache in the current logic.
https://github.com/kubewharf/godel-scheduler/blob/dcfb068e4b62f6b56f08da094e71fd1114007663/pkg/scheduler/core/pod_scheduler/pod_scheduler.go#L690-L692

Reply:
ok got it. then you can try to mock it instead of replacing it with some irrelevant names.